src/services/account/hooks/index.js

```diff
@@ -97,6 +97,7 @@ const validatePassword = (hook) => {
 						)
 					)
 				)
+				|| userForcedToChangePassword
```

Comment:
This looks like a big security risk. any user is able to set userForcedToChangePassword in the payload of his call, so this would allow any user to circumvent the security checks, and change any password via the account route.

Please consider how to solve this in a different way.

Reply:
The flag userForcedToChangePassword does not come from the request payload but it is added as additional parameter in fie src/services/user/services/ForcePasswordChange.js line 44 . So I think it is not a security threat or is it still ? You can take a look at this comment https://github.com/hpi-schul-cloud/schulcloud-server/pull/1455#discussion_r446146028 to see, what is the reason behind this flag.

Reply:
The flag is set by the forcePasswordChangeService, but that doesnt keep anyone else for setting the same flag themselfes.
hook.data contains the user input, and you have no way of checking wether the user or another server service has put in this flag.